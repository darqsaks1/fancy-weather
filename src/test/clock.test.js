// import React from 'react';
// import DayTheme from '../js/control-block/dayTheme'
// import TestRenderer from 'react-test-renderer';

//     test('Prewww', () => {
//         const component = TestRenderer.create(
//           <DayTheme 
//           setDayThemeTrue = {true }
//           setDayThemeFalse= {false }
//             dayTheme = {true}
//             />,
//         );
//         let tree = component.toJSON();
//         expect(tree).toMatchSnapshot();
//         tree.props.onClick();
//         tree.props.onClick();
//         tree = component.toJSON();
//         expect(tree).toMatchSnapshot();
//       })

import Clock from '../js/main-block/time/time' 

import React from 'react';

import TestRenderer from 'react-test-renderer';

test('Cl', () => {
  const component = TestRenderer.create(
    <Clock />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

})