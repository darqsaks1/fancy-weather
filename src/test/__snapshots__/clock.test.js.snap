// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cl 1`] = `
<div
  className="time-block textBlack"
>
  <div>
    <h2>
      <span>
        Вт
      </span>
      ,
      <span>
        2
      </span>
      <span>
        Иля
      </span>
      <span>
        22:02:05
      </span>
    </h2>
  </div>
</div>
`;

exports[`Clokc will be  1`] = `
<div
  className="time-block textBlack"
>
  <div>
    <h2>
      <span>
        Вт
      </span>
      ,
      <span>
        2
      </span>
      <span>
        Иля
      </span>
      <span>
        22:01:18
      </span>
    </h2>
  </div>
</div>
`;

exports[`Clokc will be snapshoted 1`] = `
<div
  className="time-block textBlack"
>
  <div>
    <h2>
      <span>
        Вт
      </span>
      ,
      <span>
        2
      </span>
      <span>
        Иля
      </span>
      <span>
        21:49:40
      </span>
    </h2>
  </div>
</div>
`;

exports[`Preloader changjed dom tree 1`] = `
<div
  className="time-block textBlack"
>
  <div>
    <h2>
      <span>
        Вт
      </span>
      ,
      <span>
        2
      </span>
      <span>
        Иля
      </span>
      <span>
        21:49:04
      </span>
    </h2>
  </div>
</div>
`;
